src/platform/i386/boot_comp.c

```diff
@@ -161,9 +171,11 @@ kern_boot_comp(void)
 		assert(!ret);
 	}
 
-	thd_mem  = mem_boot_alloc(1);
-	tcap_mem = mem_boot_alloc(1);
-	assert(thd_mem && tcap_mem);
+	for (i = 0; i < NUM_CPU; ++i) {
```

Comment:
nit: I don't remember what CSG says for use of `pre-increment vs post-increment` in `for` loops (though the effect is same here).
In most places we use `post-increment`.

Reply:
dito